### ISSUE TITLE
WCAG tags in rule-descriptions list inconsistent with the WCAG Success Criteria in each individual rule page

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#575]
- The following rules have inconsistent tags associated with them

Rule ID | rule-descriptions list | Individual rule page
-- | -- | --
aria-required-parent | 1.3.1 | 4.1.1, 4.1.2
frame-title | 2.4.1 | 4.1.2
link-in-text-block | 1.4.1 | 1.4.3
link-name | 1.1.1, 4.1.2,   2.4.4 | 1.1.1, 4.1.2
valid-lang | 3.1.2 | 3.1.1, 3.1.2

- The following best practice rules in [rule-desciptions](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) need to be associated with WCAG tags
    - checkboxgroup
    - empty-heading
    - frame-title-unique
    - heading-order
    - hidden-content
    - href-no-hash
    - image-redundant-alt
    - label-title-only
    - meta-viewport-large
    - radiogroup
    - region
    - scope-attr-valid
    - skip-link
    - tabindex
    - table-duplicate-name

**Comment by WilcoFiers** [comment]
Thanks for the notice. We've updated the rules pages around best practices. Hopefully this will be clearer now.

**Comment by iamrafan** [comment]
@WilcoFiers Sure, thanks. Did you mean all these issues are fixed in 3.0 version of the rules

**Comment by WilcoFiers** [comment]
Not quite. What we did is update the help pages to more clearly specify that the best practices are not part of the WCAG requirements. I think this distinction is important. I don't want us to confuse WCAG requirements with best practices, so best practice rules shouldn't have WCAG tags on them.

**Comment by iamrafan** [comment]
@WilcoFiers : Sure, I was just mentioning though the 3.0 version of help pages reflect the changes you mentioned, the 2.4 version still has WCAG tags on the.

**Comment by WilcoFiers** [comment]
Aahh like that. Apologies, that seems to be an oversight on our end. We'll get it sorted in the coming week. Thanks!

**Comment by iamrafan** [comment]
@marcysutton all the tags in the 2.6v help pages look consistent except for the following two:

- https://dequeuniversity.com/rules/axe/2.6/checkboxgroup - add best practice tag and remove Section 508 tag

-  https://dequeuniversity.com/rules/axe/2.6/frame-title-unique - remove Section 508 tag

**Comment by marcysutton** [comment]
These should be complete!

**Comment by iamrafan** [comment]
thanks
